docs/02-app/02-api-reference/05-next-config-js/images.mdx

```diff
@@ -172,13 +173,22 @@ export default function imgixLoader({ src, width, quality }) {
 }
 ```
 
-### Thumbor
+### PixelBin
 
 ```js
-// Docs: https://thumbor.readthedocs.io/en/latest/
-export default function thumborLoader({ src, width, quality }) {
-  const params = [`${width}x0`, `filters:quality(${quality || 75})`]
-  return `https://example.com${params.join('/')}${src}`
+// Doc (Resize): https://www.pixelbin.io/docs/transformations/basic/resize/#width-w
+// Doc (Optimise): https://www.pixelbin.io/docs/optimizations/quality/#image-quality-when-delivering
+export default function pixelBinLoader({ src, width, quality }) {
+  const url = new URL(src)
+  if (!url.searchParams.has('f_auto')) {
+    url.searchParams.set('f_auto', 'true')
+  }
+  url.pathname = url.pathname.replace(
+    `/original/`,
+    `/t.resize(w:${width})~t.compress(q:${quality || 75})/`
+  )
+  return url.href
+}
 }
```

Comment:
```suggestion
export default function pixelBinLoader({ src, width, quality }) {
  return `https://cdn.pixelbin.io/v2/cloudname/t.resize(w:${width})~t.compress(q:${quality || 75})/${src}?f_auto=true`
```

If you look at the other examples, they assume that `src` is a relative url, not absolute. 

Lets assume relative url here too so its easier to swap between providers.

Reply:
Updated code by assuming that `src` is a relative url, not absolute.

Use different coding approach than your suggested code.